internal/handlers/handler_test.go

```diff
@@ -558,6 +571,26 @@ func TestReadOnlyHandlers(t *testing.T) {
 			),
 		},
 
+		"HostInfo": {
+			req: types.MustMakeDocument(
+				"hostInfo", int32(1),
+			),
+			resp: types.MustMakeDocument(
+				"system", types.MustMakeDocument(
+					"currentTime", fmt.Sprintf("ISODate(%s)", nowStr),
```

Comment:
I think that's really BSON time; ISODate is how it is represented in Mongo shell.

When you run FerretDB with `make run`, you should see the diff between FerretDB and MongoDB responses. That should make data types visible.

Reply:
Got it, fixed. Thanks!